lib/motion.h.rb

```diff
@@ -68,7 +68,9 @@ def generate_bridgesupport_file
         "--64-bit"
       end
     end
-    `/Library/RubyMotion/bin/gen_bridge_metadata --format complete #{flag} --cflags '-I#{include_path} -F#{frameworks_path}' #{@header_file} > #{bridgesupport_file}`
+    Bundler.with_clean_env do
```

Comment:
It's been a while since I've done RubyMotion. Is `Bundler` guaranteed to be loaded?

Reply:
I think it's a safe assumption yes. I can deal with the fallout if there ends up being any.